src/dotnet/Chat.UI.Blazor/Components/DeleteMessageModal.razor

```diff
@@ -24,7 +24,7 @@
                     </div>
                     <div class="message-content">
                         @if (FirstAttachment != null) {
-                            var isImage = FirstAttachment.ContentType.OrdinalIgnoreCaseStartsWith("image");
+                            var isImage = FirstAttachment.Media.ContentType.OrdinalIgnoreCaseStartsWith("image");
```

Comment:
just a suggestion: how about ext method for attachment?

Reply:
Done